### DISCUSSION TITLE
I have one page with multiple form

### DISCUSSION THREAD

**Discussion by murat-yavuzer** [react-hook-form/react-hook-form#3000] [Q&A]
I have one page with multiple form. Each form is independent from other. My question is how to handle all form submitting with one button? for react hooks form v5

**Comment by bluebill1049** [comment]
you will have to gather those page data in your local state (state lib) and submit them.

**Reply by murat-yavuzer** [reply]
According to the example you have given, there is an example made on formik. Can I lose performance in such a form structure?
[for example formik](https://codesandbox.io/s/formik-multiple-form-in-one-submission-5qmt2)

**Reply by bluebill1049** [reply]
have a look at the following video tutorial: 

https://www.youtube.com/watch?v=evDxlqnsxXc
https://www.youtube.com/watch?v=CeAkxVwsyMU

they both don't have an impact on perf.

**Reply by murat-yavuzer** [reply]
@bluebill1049 Thanks bro, the information you gave is useful to me
